Networking/NetworkingTests/Remote/ProductsRemoteTests.swift

```diff
@@ -95,4 +95,45 @@ class ProductsRemoteTests: XCTestCase {
 
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
+
+    // MARK: - Search Products
+
+    /// Verifies that searchProducts properly parses the `products-load-all` sample response.
+    ///
+    func testSearchProductsProperlyReturnsParsedProducts() {
+        let remote = ProductsRemote(network: network)
+        let expectation = self.expectation(description: "Load All Products")
```

Comment:
Maybe change the expectation description to "Load search results for Products" or something similar?